editor/gui/credits_roll.cpp

```diff
@@ -88,13 +89,22 @@ String CreditsRoll::_build_string(const char *const *p_from) const {
 	return sb.as_string();
 }
 
+void CreditsRoll::_visibility_changed() {
+	if (!is_visible()) {
+		set_process_internal(false);
```

Comment:
Shouldn't you also disable input?
(although it seems to be unnecessary 🤔)
EDIT:
Ah, you are freeing the node. Then setting process should not be needed either.

Reply:
Correct, but I agree that it still should disable it anyways.